packages/caip/src/caip19/caip19.test.ts

```diff
@@ -33,8 +33,8 @@ describe('caip19', () => {
       expect(result).toEqual('cosmos:cosmoshub-4/slip44:118')
     })
 
-    it('can return ibc asset caip 19 for cosmos', () => {
-      const chain = ChainTypes.Cosmos
+    it('can return ibc asset caip 19 for osmosis', () => {
```

Comment:
We should add the applicable corresponding cosmos caip19 tests as well. There is no ibc cosmos test case or osmosis mainnet test case

I am not sure if the cw variants are applicable to cosmos or osmosis specific